apypie/action.py

```diff
@@ -85,3 +85,19 @@ def filter_empty_params(self, params=None):
                 raise InvalidArgumentTypesError
         else:
             return {}
+
+    def prepare_params(self, input_dict):
```

Comment:
It feels like this could use a docstring

Reply:
I've added one, what do you think?